https://issues.redhat.com/browse/ACM-10023 (#6267)
open-cluster-management/rhacm-docs: governance/policy_operator.adoc

The operator policy controller allows you to monitor and install Operator Lifecycle Manager (OLM) operators across your clusters. Use the operator policy controller to monitor the health of various pieces of the operator and to specify how you want to automatically handle updates to the operator. You can also distribute an operator policy to managed clusters by using the governance framework and adding the policy to the `policy-templates` field of a policy on the hub cluster.

[#pre-req-policy-operator]
== Prerequisites

> * You must install OLM on your cluster. See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/operators/index#olm-adding-operators-to-a-cluster[Adding Operators to a cluster] in the _{ocp} documentation_.

Reviewer: Optional: 

I think links are easier to click and resources are easier to find if links are at the end of the sentence. So, consider rewording to something like: 

```suggestion
* You must install OLM on your cluster.  In the _{ocp} documentation_, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/operators/index#olm-adding-operators-to-a-cluster[Adding Operators to a cluster].
```

    - quay-operator.v3.10.3
----

[#policy-operator-yaml-table]

> == Operator policy YAML table

Reviewer: The content in this table is great! 

For future work, we might want to consider if some of this content would be better in a concept doc/module. I'm used to seeing tables being fairly brief and concise. I don't think any of this content ought to be removed, but perhaps some of it should be incorporated throughout our other docs on the operator policy yaml.

Author: @jc-berger thanks for your point of view Jake. I'm open to having a convo about improving this. Throughout the Governance doc, we usually provide an empty policy sample so that the user can see how it should be indented. Then the table is used to list the parameters and the functions. Sometimes developers will add the description as  a comment in the policy. Using the table was my first solution, and i'm open to evolving the solution. Let's bring this up in the Doc arch call

|===
| Field | Optional or required | Description

| `apiVersion`
| Required
| Set the value to `policy.open-cluster-management.io/v1beta1`.

| `kind`
| Required
| Set the value to `OperatorPolicy` to indicate the type of policy.

| `metadata.name`
| Required
| The name for identifying the policy resource.

| `spec.remediationAction`
| Required

> | If the `remediationAction` for the operator policy is set to `enforce`, the controller creates resources on the target managed cluster to install the operator and approves updates based on the versions specified in the policy.

Reviewer 2: I think `...the controller creates resources on the target managed cluster to trigger OLM to install the operator...` may work better?

> | If the `remediationAction` for the operator policy is set to `enforce`, the controller creates resources on the target managed cluster to install the operator and approves updates based on the versions specified in the policy.

Author: ```suggestion
| If the `remediationAction` for the operator policy is set to `enforce`, the controller creates resources on the target managed cluster to communicate to OLM to install the operator and approve updates based on the versions specified in the policy.
```

+
If the `remediationAction` set to `inform`, the controller only reports the status of the operator, including if any upgrades are available.

| `spec.operatorGroup`
| Optional

> | By default, if the `operatorGroup` field is not specified, the controller generates an `AllNamespaces` type OperatorGroup in the same namespace as the subscription, if supported. This value is generated by the operator policy controller.

Reviewer 3: ```suggestion
| By default, if the `operatorGroup` field is not specified, the controller generates an `AllNamespaces` type OperatorGroup in the same namespace as the subscription, if supported. This resource is generated by the operator policy controller.
```

| `spec.subscription`
| Required
| Define the configurations to create an operator subscription. You must add information in the following fields to create an operator subscription:
+
- `channel`
- `name`
- `namespace`
- `source`
- `sourceNamespace`

| `subscriptions.installPlanApproval`
| Required

> | If the `installPlanApproval` field is set to `Manual`, you must manually patch the `.spec.approved` field in the relevant `InstallPlan` resource to proceed with the installation process. The `InstallPlan` resource for an operator is generated after the subscription is created by the controller, and represents the intention to install a specific version of the desired operator.

Reviewer 3: ```suggestion
| If the `installPlanApproval` field is set to `Manual`, and the `spec.versions` field is empty, you must manually patch the `.spec.approved` field in the relevant `InstallPlan` resource to proceed with the installation process. The `InstallPlan` resource for an operator is generated after the subscription is created by the controller, and represents the intention to install a specific version of the desired operator.
```

| `spec.versions`
| Optional

> | Declare which versions of the operator are compliant. If the field is empty, any version running on the cluster is considered compliant. If the field is not empty, the version on the managed cluster must match one of the versions in the list for the policy to be compliant.

Reviewer 3: ```suggestion
| Declare which versions of the operator are compliant. If the field is empty, any version running on the cluster is considered compliant. If the field is not empty, the version on the managed cluster must match one of the versions in the list for the policy to be compliant. If the policy is set to `enforce` and the list is not empty, the versions listed here are approved by the controller on the cluster.
```

Author: @JustinKuli thanks Justin, I edited your suggestion to remove future tense. Thanks for the suggestion!
